### ISSUE TITLE
Staking on Ubuntu 20.04 VPS - sycn complete, 8 hours staking, no rewards

### ISSUE THREAD

**Issue by skippa** [thetatoken/guardian-mainnet-guide#49]
I setup a CLI guardian node using this guide https://docs.thetatoken.org/docs/running-a-guardian-node-through-command-line - staked 1000 THETA - everything seems ok:  `sync = true` when I run `./thetacli query guardian`

My mobile wallet shows that my stake is in place.  My node even appears on https://guardianmonitor.io/ but shows that its OFFLINE/NEW NODE (address is 0x879283af95d925be6db63d131b01f203114c117c).  Theta Explorer shows its a guardian node https://explorer.thetatoken.org/account/0x879283af95d925be6db63d131b01f203114c117c/

I've read on other issues that some folks have had to delete the `snapshot` and `db` folders.  Is this safe to do?  Or should I withdraw my stake BEFORE doing this?  Don't wanna burn any THETA by getting this wrong ;)

Will post a couple of screen grabs from the node log console (it moves fast!)

**Comment by skippa** [comment]
![Screenshot_20210128_063558_com sonelli juicessh](https://user-images.githubusercontent.com/5898372/106052804-c4550800-6135-11eb-827c-020f1d731784.jpg)
![Screenshot_20210128_063554_com sonelli juicessh](https://user-images.githubusercontent.com/5898372/106052818-c74ff880-6135-11eb-95fa-c6d3d2fdec9d.jpg)
![Screenshot_20210128_063548_com sonelli juicessh](https://user-images.githubusercontent.com/5898372/106052823-c7e88f00-6135-11eb-8646-02e4c9660da1.jpg)

**Comment by skippa** [comment]
So I get a bunch of these - which to me seems healthy

`[2021-01-28 21:18:32]  INFO [mempool] Insert tx, tx.hash: 0x2e25c8eafe792ea8c14036e6cf45abf1edacc058bcdf8ddf4e983f98885b9c2a
[2021-01-28 21:18:32]  INFO [mempool] Insert tx, tx.hash: 0x31629d6bc3ddb9ccd8eb4f77c72b42530b2218a60903f8eb97756d89a85575ea
[2021-01-28 21:18:32]  INFO [mempool] Insert tx, tx.hash: 0xdfcbd000ed9a314b13af6e860cbfdc40b77dbdf5aae7b2f4e9b6c942fcf79add`

But I also get a lot of these
`[2021-01-28 21:18:34]  WARN [ledger] validateSourceAdvanced failed on 0x8aE3e4b038B4Db99a3c0087139cd0eb772AB2123: Result{code:100002, message:ValidateInputAdvanced: Got 6, expected 7. (acc.seq=6)}`

And these
`[2021-01-28 21:24:31]  WARN [p2p] Stopping connection, local: 195.154.136.47:30001, remote: 96.38.171.97:54755
[2021-01-28 21:24:31]  WARN [p2p] Stopping duplicated peer: 0x44DD17CE4cA4897C70CB73d1e89CE41B61bA6B58`

Can anyone shed any light on exactly how to determine what's going on with the node?

**Comment by skippa** [comment]
I still couldn't get this to work. I did notice that `syncing = true` then thought perhaps the problem was that I may have staked before `syncing = false`. Block height seemed to be a few thousand behind that reported on theta block explorer. So I'm gonna withdraw, then start from scratch.

Can any devs provide some insight here? I would have thought that staking before the node is ready would a) not be allowed or b) the node would eventually sync up. 

Is there a discord / slack / telegram I can join for support?

**Comment by skippa** [comment]
FYI I unstaked my Theta and started from scratch. I let it sync overnight so not sure exactly how long it took. But the guide should be updated because I think when it says about 10 mins to sync, and it doesn't happen, you think there's something wrong. Also, it appears that if you stake too early before sync is complete, it never catches up or gives rewards.

Anyways, this is all guesswork, so if a dev can chime in to give us some insight as to what happened here it might help others 👌

**Comment by texmason** [comment]
@Skippa were you ever able to start staking?   I have to do a lot of tweaking to get SYNCING but even after that, I haven't had success staking.

**Comment by skippa** [comment]
Nope, I found it too unreliable. It did run successfully for about 24 hours, but then fell behind on the blocks. Deleted db and got latest snapshot again, sync'd again after a couple of hours, but wasn't giving rewards and fell behind on blocks again. Ended up using gpool.io instead.

My sense is that, for whatever reason, Linux development has taken a bit of a backseat at this early stage of the project. The Windows and Mac GN implementations seem to work better. It's possible I just don't have enough CPU (although my spec meets requirements) but logs don't tell you anything along those lines.

**Comment by texmason** [comment]
I think your right on the money. It sucks because I was really excited about this project.  Mining Theta in the cloud

**Comment by skippa** [comment]
I just think we're so early that the devs are just doing what's needed to bootstrap the network, I know new Linux releases are coming.  Mainnet 3.0 will bring a bunch of improvements too. Hang in there amigo! BTW using qemu to run a Windows VM with an Edge Node is keeping me happy while I wait for better Linux support. You could probably do the same to run a Windows GN if you're too impatient ;)

**Comment by ghost** [comment]
Where do you get the 'latest snapshot'. I'm running theta cli on unbntu server and watching it sync. Looks like it will take days. I just did a fresh install. I'm wondering if this is practical.

**Comment by Garitty7612** [comment]
Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thetanetworking. They'll be happy to resolve your issue. Cheers.

**Comment by ghost** [comment]
Garitty7612 is a scammer. Beware!!!!

**Comment by ghost** [comment]
Where is REAL theta support. github seems useless.

**Comment by hg9679** [comment]
> I just think we're so early that the devs are just doing what's needed to bootstrap the network, I know new Linux releases are coming. Mainnet 3.0 will bring a bunch of improvements too. Hang in there amigo! BTW using qemu to run a Windows VM with an Edge Node is keeping me happy while I wait for better Linux support. You could probably do the same to run a Windows GN if you're too impatient ;)

Can you answer some of this question https://github.com/thetatoken/guardian-mainnet-guide/issues/90

**Comment by ghost** [comment]
Thanks for responding. I can't answer your question. The problem is that I'm running a server with no desktop, trying to do everything on the command line. But I *could* install the desktop version of Ubuntu, I guess. I ran a guardian node for the month of January with no real problem, but now nothing works. I'll keep waiting for an update or install the desktop. Thank you.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
